packages/lib/src/test_utils/utils.ts

```diff
@@ -1,18 +1,39 @@
-export const mockEnvironmentVariables = () => {
+type MockEnvironmentVariables = {
+  /**
+   * Cleans up the mocked environment variables.
+   */
+  cleanup: () => void;
+  /**
+   * Mocks an environment variable.
+   * @param variable - The environment variable to mock.
+   * @param value - The value to set the environment variable to.
+   */
+  mockEnv: (variable: string, value: string) => void;
+};
+
+/**
+ * Creates mock environment variables for testing purposes.
+ * @returns An object with `cleanup` and `mockEnv` functions.
+ */
+export const mockEnvironmentVariables = (): MockEnvironmentVariables => {
```

Comment:
I think we could switch to Vitest's [`stubEnv`](https://vitest.dev/api/vi.html#vi-stubenv)

Maybe you could add a `@deprecated` tag here?

Reply:
Removed it and replaced it all with stubEnv!